File: src/main/java/com/ivaneye/intellijweb2/respository/OrdersRespository.java
package com.ivaneye.intellijweb2.respository;

import com.ivaneye.intellijweb2.mapper.OrdersMapper;
import com.ivaneye.intellijweb2.param.OrdersParam;

/**
 * <p>
 *  Mapper 接口
 * </p>
 *
 * @author wangyifan
 * @since 2018-02-12
 */
public interface OrdersRespository extends OrdersMapper {

    void insertAndGet(OrdersParam param);
}

File: src/main/kotlin/com/ivaneye/intellijweb2/respository/OrdersRespository.kt
package com.ivaneye.intellijweb2.respository

import com.ivaneye.intellijweb2.mapper.OrdersMapper
import com.ivaneye.intellijweb2.param.OrdersParam

/**
 * Created by wangyifan on 18-2-19.
 */
interface OrdersRespository : OrdersMapper {
    fun insertAndGet(param: OrdersParam)
}